config_test.go

```diff
@@ -4,9 +4,33 @@ import (
 	"path/filepath"
 	"testing"
 
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/send"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
 )
 
+const (
+	testDir      = "config_test"
+	testSettings = "evg_settings.yml"
+)
+
+// TestConfig creates test settings from a test config.
+func testConfig() *Settings {
+	file := filepath.Join(FindEvergreenHome(), testDir, testSettings)
+	settings, err := NewSettings(file)
+	if err != nil {
+		panic(err)
+	}
+
+	if err = settings.Validate(); err != nil {
+		panic(err)
+	}
```

Comment:
these panics (even in tests) seem less than awesome but maybe its alright?

Reply:
This copy of the function will go away with the refactor, so I think it's find to keep here